### ISSUE TITLE
byDay should be an array

### ISSUE THREAD

**Issue by domfennell** [RunTogether/opendata#11]
I think I'm right in saying that `eventSchedule[].byDay` is currently a string when the OpenActive Specification indicates [here](https://openactive.io/modelling-opportunity-data/#:~:text=schema%3AbyDay,BYDAY%20rule) that it should be an array.

**Comment by shervinw** [comment]
Yes @domfennell this is causing a few issues for us, so would be great to get it resolved.
